src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx

```diff
@@ -75,21 +83,18 @@ export const MobileArticleVideoPlayerContent: React.FC<MobileArticleVideoPlayerP
 			recordVideoEvent(VIDEO_RECORD_EVENTS.JW_PLAYER_PLAYING_VIDEO);
 			return;
 		}
-	}, [onScreen, adComplete]);
+	}, [onScreen, adEngineComplete]);
 
 	return (
 		<>
 			<div ref={ref} className={clsx(styles.mobileArticleVideoTopPlaceholder, isScrollPlayer && `is-on-scroll-active`)}>
 				<MobileArticleVideoWrapper isScrollPlayer={isScrollPlayer}>
-					{adComplete && (
+					{jwpAdsSetupComplete.strategyRulesEnabled && (
 						<>
-							<JwPlayerWrapper
+							<StrategyRulesWrapper
```

Comment:
I think we should still render JwPlayerWrapper if `strategyRulesEnabled` is `false`, right?

Reply:
right, I initially thought we should substitute one with the other. But that's indeed what nandy's PR suggesting. Let me fix it.